internal/commands/build_test.go

```diff
@@ -103,6 +103,16 @@ func testBuildCommand(t *testing.T, when spec.G, it spec.S) {
 					h.AssertNil(t, command.Execute())
 					h.AssertContains(t, outBuf.String(), "Builder 'my-builder' is trusted")
 				})
+				when("a lifecycle-image is provided", func() {
+					it("ignoring the mentioned lifecycle image, going with default version", func() {
+						mockClient.EXPECT().
+							Build(gomock.Any(), EqBuildOptionsWithLifecycleImage("index.docker.io/library/some-lifecycle-image:latest")).
+							Return(nil)
+
+						command.SetArgs([]string{"--builder", "my-builder", "image", "--lifecycle-image", "some-lifecycle-image"})
+						h.AssertNil(t, command.Execute())
```

Comment:
Thank you @jjbustamante for the suggestion. I will make the changes as per your request.

Reply:
@cgoxo you may have encountered this already, but just in case, you may find the content in https://github.com/buildpacks/pack/blob/main/DEVELOPMENT.md helpful as you work through the process. If you have any questions, please don't hesitate to ask!

Reply:
@natalieparellano! The `DEVELOPMENT.MD` file is super helpful to me. Thank you for making these docs beginner friendly.